src/components/views/rooms/RoomListPanel/RoomListItemView.tsx

```diff
@@ -94,7 +103,7 @@ export const RoomListItemView = memo(function RoomListItemView({
                         <div className="mx_RoomListItemView_roomName" title={vm.name}>
                             {vm.name}
                         </div>
-                        <div className="mx_RoomListItemView_messagePreview">{vm.messagePreview}</div>
+                        {messagePreview}
```

Comment:
```suggestion
                        {vm.messagePreview && <div className="mx_RoomListItemView_messagePreview" title={vm.messagePreview}>
                {vm.messagePreview}
            </div>}
```

(indentation in suggestion :()

Reply:
imo, it's easier to read the component when the jsx is written alongside the other jsx node